tests/test_other.py

```diff
@@ -503,11 +494,7 @@ def check_makefile(configuration, dirname):
       ]
       for test_dir, output_file, cmake_args in cases:
         cmakelistsdir = path_from_root('tests', 'cmake', test_dir)
-        # Create a temp workspace folder
-        tempdirname = tempfile.mkdtemp(prefix='emscripten_test_' + self.__class__.__name__ + '_', dir=TEMP_DIR)
```

Comment:
Are we okay to throw these dynamic prefixes away?

Reply:
It'd be pretty easy to modify `temp_directory` to include them; my initial class before I spotted `temp_directory` was setup like:

```python
  def __init__(self, test_runner, **kwargs):
    self.orig_cwd = None

    kwargs.setdefault('prefix', 'emscripten_test' + test_runner.__class__.__name__ + '_')
    kwargs.setdefault('dir', TEMP_DIR)
    self.name = mkdtemp(**kwargs)
```

so you'd do:

```python
with InTempDirectory(self):
    ....
```